lib/routes/routes_generator.dart

```diff
@@ -0,0 +1,18 @@
+import 'package:flutter/material.dart';
+import 'package:workoutpersonalizer_frontend/routes/routes.dart';
+import 'package:workoutpersonalizer_frontend/ui/creators_corner_page.dart';
+import 'package:workoutpersonalizer_frontend/ui/dashboard_page.dart';
+import 'package:workoutpersonalizer_frontend/ui/my_workouts_page.dart';
+
+class RouteGenerator {
+  static Route<dynamic> generateRoute(RouteSettings settings) {
+    switch (settings.name) {
+      case routeDashboard:
+        return MaterialPageRoute(builder: (_) => DashboardPage());
```

Comment:
Just to remove redundancy but maybe we can set the page in the switch and only call 
`return MaterialPageRoute(builder: (_) =>` once?